internal/versions/versions.go

```diff
@@ -73,6 +81,10 @@ func (h *handler) ListSupportedOpenshiftVersions(ctx context.Context, params ope
 }
 
 func (h *handler) GetReleaseImage(openshiftVersion string) (pullSpec string, err error) {
+	openshiftVersion, err = h.GetSupportedKeyVersionFormat(openshiftVersion)
```

Comment:
Please use a different convention in case we would still want to use the same given `openshiftVersion`
```suggestion
	versionKey, err = h.GetSupportedKeyVersionFormat(openshiftVersion)
```